app.py

```diff
@@ -41,20 +40,50 @@ def _benchmark_import_time() -> Dict[str, float]:
     }
 
 
-def _benchmar_flows() -> Dict[str, None]:
-    # TODO
+def _benchmark_avg_flow_time() -> Dict[str, float]:
+    """Benchmark on a simple flow operation.
+    
+    Reurns:
+        A dict mapping of import time in seconds as float number.
+    """
     fs = [
-        Flow(),
         Flow().add(),
         Flow().add().add(),
-        Flow().add().add(needs='gateway')]
-
+        Flow().add().add().add(),
+        Flow().add().add().add(needs='gateway')
+    ]
+    st = time.perf_counter()
     for f in fs:
-        f.post('/', (Document() for _ in range(10000)))
-        f.post('/', (Document(blob=sp.coo_matrix([0, 0, 0, 1, 0])) for _ in range(10000)))
+        with f:
+            f.post(on='/test', inputs=Document())
```

Comment:
I think we could average by flowing more `requests` per `Flow` and not a single `Document`?

Reply:
Specially with `dynamic routing` the first request should be slower because sockets are open